### PR TITLE
Updated Onkyo documentation inline with code changes

### PR DESCRIPTION
**Pull request by winterscar**
## Proposed change

I have updated the Onkyo component. This clarifies the new zones key and removes info about auto-discovery.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/33694
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/onkyo.markdown [hunk 1]
```diff
@@ -86,15 +97,15 @@ List of source names:
 If your source is not listed above, and you want to figure out how to format that source name so you can map its entry, you can use the `onkyo-eiscp` Python module to discover the exact naming needed. First, change your receiver's source to the one that you need to define, and then run:
 
 ```bash
-onkyo --host 192.168.0.100 source=query
+eiscp_monitor --host 192.168.0.100 source=query
```
Comment (frenck) [inline source/_integrations/onkyo.markdown:96]: How would a Home Assistant or Home Assistant Core on Docker user run this?

File: source/_integrations/onkyo.markdown [hunk 2]
```diff
@@ -25,15 +25,21 @@ media_player:
...
 
 {% configuration %}
 host:
-  description: IP address of the device. Example:`192.168.1.2`. If not specified, the platform will load any discovered receivers.
-  required: false
+  description: IP address of the device. Example:`192.168.1.2`.
+  required: true
   type: string
+port:
+  description: The port of the receiver. Usually 60128. Only set if you need to change the default.
```
Comment (frenck) [inline source/_integrations/onkyo.markdown]: Move the port to the `default: ` key.

File: source/_integrations/onkyo.markdown [hunk 3]
```diff
@@ -25,15 +25,21 @@ media_player:
     name: receiver
     sources:
       pc: 'HTPC'
+    zones:
+      - zone1
 ```
 
- If your receiver has second or third zone’s available, they are displayed as additional media players with the same functionality as the main zone.
+ If your receiver has second or third zone’s available, they are displayed as additional media players with the same functionality as the main zone, provided you add them to your configuration under `zones`. Zones 1-4 are supported.
```
Comment (frenck) [inline source/_integrations/onkyo.markdown]:
```suggestion
 If your receiver has second or third zones available, they are displayed as additional media players with the same functionality as the main zone, provided you add them to your configuration under `zones`. Zones 1-4 are supported.
```

File: source/_integrations/onkyo.markdown [hunk 4]
```diff
@@ -25,15 +25,21 @@ media_player:
     name: receiver
     sources:
       pc: 'HTPC'
+    zones:
```
Comment (frenck) [inline source/_integrations/onkyo.markdown]: We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

File: source/_integrations/onkyo.markdown [hunk 5]
```diff
@@ -52,6 +58,11 @@ sources:
   description: A list of mappings from source to source name. Valid sources can be found below. A default list will be used if no source mapping is specified.
   required: false
   type: list
+zones:
+  description: Any secondary zones you wish to see in Home Assistant. Each zone listed here will show up as a media player in it's own right.
```
Comment (frenck) [inline source/_integrations/onkyo.markdown]: Home Assistant is kinda implied already 😉 

```suggestion
  description: Any secondary zones you wish to see. Each zone listed here will show up as a media player in its own right.
```

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks for the PR @winterscar!

I've reviewed it and left some comments and suggestions. Could you please take a look? Thanks! 👍

File: source/_integrations/onkyo.markdown (hunk 1 above)
Comment (winterscar) [reply source/_integrations/onkyo.markdown:96]: This gets added as part of the pyeiscp library, same way onkyo got added as part of the onkyo-eiscp library. You should be able to use docker exec -it /bin/bash home-assistant then run the tool. 
Should I add that in?

**Comment by winterscar** [comment]
Thanks for reviewing @frenck! I've made the changes as per your requests.

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by klaashoekstra94** [comment]
I know this PR is already approved, but I just noticed how in the example the name is missing, although I do think it is required because the host is now also required.

Also, as seen in the source code [here](https://github.com/home-assistant/core/blob/b1abe5918adcbe25948411185cb16fb13da94a5d/homeassistant/components/onkyo/media_player.py#L44), the values for the zones list should exactly equal `zone2`, `zone3`, or `zone4`. Should we add this to the documentation as well?

File: source/_integrations/onkyo.markdown (hunk 1 above)
Comment (frenck) [reply source/_integrations/onkyo.markdown:96]: OS users don't have that ability.

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Pull request closed without merging**